hazelcast/src/main/java/com/hazelcast/config/SerializationConfig.java

```diff
@@ -63,6 +63,8 @@
 
     private Set<ClassDefinition> classDefinitions;
 
+    private JavaSerializationFilterConfig javaSerializationFilterConfig;
```

Comment:
This is where a default config should be created IF we want to have any default filtering.